admin_guide/cluster_management.rst

```diff
@@ -487,23 +493,85 @@ To set non-sensitive libpq connection parameters to be used for all node connect
 
   ALTER DATABASE foo
   SET citus.node_conninfo =
-    'sslrootcert=/path/to/citus.crt sslmode=verify-full';
+    'sslrootcert=/path/to/citus-ca.crt sslcrl=/path/to/citus-ca.crl sslmode=verify-full';
+
+There is a whitelist of parameters that the GUC accepts, see the :ref:`node_conninfo <node_conninfo>` reference for details. As of Citus 8.1, the default value for node_conninfo is ``sslmode=require``, which prevents unencrypted communication between nodes. If your cluster was originally created before Citus 8.1 the value will be ``sslmode=prefer``. After setting up self-signed certificates on all nodes it's recommended to change this setting to ``sslmode=require``.
+
+After changing this setting it is important to reload the postgres configuration. Even though the changed setting might be visible in all sessions, the setting is only consulted by Citus when new connections are established. When a reload signal is received citus marks all existing connections to be closed which causes a reconnect after running transactions have been completed.
+
+.. code-block:: postgresql
+
+  SELECT pg_reload_conf();
```

Comment:
Is this something that needs to be propagated through `run_command_on_workers()`?

Reply:
Good question. That depends a bit what you mean. If you only change `citus.node_conninfo` on one node, there's no reason to reload the configuration on the other nodes. Often you probably want to change the configuration on all nodes. So then it would make sense to do something like:
```sql
ALTER SYSTEM SET citus.node_conninfo =
    'sslrootcert=/path/to/root.crt sslcrl=/path/to/root.crl sslmode=verify-full';
select pg_reload_conf();
select run_commands_on_workers($$
ALTER SYSTEM SET citus.node_conninfo =
    'sslrootcert=/path/to/root.crt sslcrl=/path/to/root.crl sslmode=verify-full';
$$)
select run_commands_on_workers($$
select pg_reload_conf();
$$)
```

This assumes these files are in the same place on all nodes though.

I think it's fine to leave this like this for the initial version of this change.